### DISCUSSION TITLE
HTTP Error! Please help

### DISCUSSION THREAD

**Discussion by maryadejoh** [IBAHCM/RPiR#486] [Archive] [answered]
Hi @IBAHCM/rpir-instructors. I keep getting a particular error when i run devtools::install()
I had to delete the whole package and repository thinking it was only limited to the project but tried it on the series03 practical and got the same error.

This is what I get anytime I run devtools::install():
Error: HTTP error 401.
  Bad credentials

  Rate limit remaining: 55/60
  Rate limit reset at: 2022-01-21 13:10:06 UTC

PS: Forgot on click on "start discussion" yesterday

**Comment by richardreeve** [comment, marked as answer]
Hi @maryadejoh - it looks like something has gone wrong with your login to GitHub from RStudio - you probably need to recreate a PAT - the instructions are here - https://ibahcm.github.io/RPiR/articles/pages/install_github.html#add-a-pat - and then try again.

**Reply by maryadejoh** [reply]
Thanks, Richard. I ended up deleting the package and uninstalling my Rstudio, Rtools and git and reinstalled. I basically started from scratch using the installation guide and I am not getting the errors anymore.
It was such a nightmare, but everything seems to be fine now.
Thanks.

**Reply by richardreeve** [reply]
Sorry to hear that. RStudio is not the most reliable thing unfortunately, but usually it's just the PAT rather than anything more fundamental. Glad you got it working anyway.
